Kajabi/sage-lib
# [SAGE-278] Button - add loader to React version like in Rails

## Description
Add loading state to react buttons

## Screenshots
<img width="177" alt="CleanShot 2022-09-01 at 13 22 27@2x" src="https://user-images.githubusercontent.com/791670/187975199-e429bb32-0f0b-4049-80dc-03408feab1e2.png">

## Testing in `sage-lib`
Navigate to Loading Button Story: http://localhost:4100/?path=/story/sage-button--loading
1. Confirm when clicked loading icon shows
3. Confirm proper a11y on button
4. Compare to rails loading button: http://localhost:4000/pages/component/button?tab=preview


## Testing in `kajabi-products`


## Related
[SAGE-278](https://kajabi.atlassian.net/browse/SAGE-278)
